src/component/LoginForm.js

```diff
@@ -9,19 +10,31 @@ class LoginForm extends React.Component {
       password: ''
    };
 
+   componentDidMount() {
+
+      let arr = [];
+      arr = JSON.parse(localStorage.getItem('user'));
```

Comment:
@RajanTank create a seprate function to get and set value from localStorage and pass the key, value as params